src/sparql/update/remove-duplicate-synonyms.ru

```diff
@@ -0,0 +1,36 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+DELETE {
+ ?cls ?synonym ?value .
+    ?x a owl:Axiom ;
+      owl:annotatedSource ?cls ;
+      owl:annotatedProperty ?synonym ;
+      owl:annotatedTarget ?value ;
+      ?p ?z .
+}
+INSERT {
+ ?cls oboInOwl:hasExactSynonym ?value .
+    ?x a owl:Axiom ;
+      owl:annotatedSource ?cls ;
+      owl:annotatedProperty oboInOwl:hasExactSynonym ;
```

Comment:
This turns every synonym regardless of what it originally was to exact. In particular, if synonym 1 is broad, and synonym 2 is narrow, both will be turned into exact.

Reply:
Yes, but it only changes the cases when they have the same value.

```
25  ?cls ?synonym ?value .
26  ?cls ?synonym2 ?value .
```

This decision came from the meeting https://github.com/obophenotype/uberon/issues/2305#issuecomment-1054564265

Reply:
This seems like a very odd decision to make. Imagine the case oboInOwl:hasRelatedSynonym oboInOwl:hasNarrowSynonym - it seems wrong to change them both to exact!

Reply:
Ah, understood now. In this case, it still has some problems with duplicate scoped synonyms.

Reply:
How many? Maybe we can fix them manually.

Reply:
It's about 15 cases.

Reply:
Cool. Would you be ok to fix them manually?

broad/related --> broad
narrow/related --> narrow
broad/narrow --> try to understand, if unclear: close

Reply:
if its 15 cases, I would rather fix them manually on a case to case basis. Happy to look into it if you open a ticket and list these 15 terms down - happy for you to assign me :) thanks

Reply:
no worries, Shawn. I did the modifications in this commit https://github.com/obophenotype/uberon/pull/2332/commits/0d0530a0e10798f3230969d1180daf6ec8f79830

Reply:
will run a robot diff again and get it ready for review :)

Reply:
it's better if you check the diff in Git Desktop so you have the context 😄

Reply:
ah wanted to do the diff in robot diff cause its super big changes, but it seems like you can actually let the diff load here. It's mainly for others to confirm that the exacts are actually exact synonyms and no mistakes were made, but totally get how its actually easier to see here than in robot diff at times

Reply:
No matter how big the diffs are, you can always open them on your machine using Git Desktop. Here on the web, that is not possible.

The commit I commented here has only the 15 cases with duplicated synonym scope that are not exact; that's why you can see them.

Reply:
Oh i meant i wanted to see the final product compared to master to see how this PR would change the ontology